code/citadel/cit_guns.dm

```diff
@@ -719,10 +719,10 @@ obj/item/projectile/bullet/c10mm/soporific
 	if((blocked != 100) && isliving(target))
 		var/mob/living/L = target
 		L.blur_eyes(6)
-		if(L.staminaloss >= 60)
-			L.Sleeping(250)
+		if(L.getStaminaLoss() >= 50)
```

Comment:
........................

Reply:
Oh god I fucked it up

Reply:
...wait, did I? It didn't seem to work at all with just `staminaloss`, at least that's what I was told when a bunch of people on the live server tried it.

Reply:
Why doesn't it just inject soporific to the mob?

Reply:
Because it uses the same code as sniper rifle soporifics rather than being a syringe thingy.

Reply:
...unless this is about the fact that it was nerfed to needing three shots before it was buffed again to two shots here because it's so useless and antags would rather kill their victim instead of trying to put them to sleep.